src/Behavioral.Automation/FluentAssertions/Assert.cs

```diff
@@ -50,10 +50,10 @@ public static void ShouldBecome<T>(this Func<T> predicate, T value, AssertionBeh
             }
         }
 
-        public static void ShouldBecome<T>(Func<T> predicate, T value, string message, bool direction = true)
+        public static void ShouldBecome<T>(Func<T> predicate, T value, string message, bool direction = true, int attempts = DefaultAttempts)
         {
 
-            for (int index = 0; index < Attempts; index++)
+            for (int index = 0; index < attempts; index++)
```

Comment:
The lines 56 and 76 request the loop to go over the different number of cycles. Is there any reason for that?

Reply:
In assert.ShouldBe one attempt is performed before a cycle so we need a lesser amount of iterations

Reply:
Accepted.